contracts/integrations/test/framework-unit-tests/function_assertion_test.ts

```diff
@@ -0,0 +1,134 @@
+import {
+    blockchainTests,
+    constants,
+    expect,
+    filterLogsToArguments,
+    getRandomInteger,
+    hexRandom,
+} from '@0x/contracts-test-utils';
+import { BigNumber, StringRevertError } from '@0x/utils';
+import { TransactionReceiptWithDecodedLogs } from 'ethereum-types';
+
+import { artifacts, TestFrameworkContract, TestFrameworkEventEventArgs, TestFrameworkEvents } from '../../src';
+import { FunctionAssertion, Result } from '../utils/function_assertions';
+
+const ZERO = constants.ZERO_AMOUNT;
+const MAX_UINT256 = constants.MAX_UINT256;
+
+blockchainTests.resets('FunctionAssertion Unit Tests', env => {
```

Comment:
I like these tests much better 👍

Reply:
I thought (and hoped) that would be the case :)